components/camel-cxf/camel-cxf-soap/src/main/docs/cxf-component.adoc

```diff
@@ -567,7 +567,13 @@ and setDefaultBus properties from spring configuration file.
 
 == Examples
 
-=== How to create a simple CXF service with POJO data format
+=== How to consume a message from a Camel CXF endpoint in POJO data format
```

Comment:
```suggestion
=== Consume a message from a Camel CXF endpoint in POJO data format
```

Shorter & same meaning

Reply:
I've used this because it is used in other headers as well.